pages/movies/[id].js

```diff
@@ -32,11 +30,24 @@ export default function MovieDetailPage({ movies }) {
             alt={`movieposter of ${movieDetail.title}`}
           />
         </MovieImageContainer>
+
+        <WrapButton>
+          <Button
+            type="button"
+            aria-label="Add/Remove Toggle Button"
```

Comment:
I think the `aria-label` should be a bit more descriptive, otherwise the user does not know what he/she toggles with that button. Maybe something like "Add or remove movies from favorite list"